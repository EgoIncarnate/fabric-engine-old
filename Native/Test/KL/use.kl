use UnknownType;
use UnknownPlugin, AnotherUnknownType;
use AnotherUnknownPlugin;
