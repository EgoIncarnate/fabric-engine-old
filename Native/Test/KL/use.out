1:5: error: no registered type or plugin named 'UnknownType'
2:5: error: no registered type or plugin named 'UnknownPlugin'
2:20: error: no registered type or plugin named 'AnotherUnknownType'
3:5: error: no registered type or plugin named 'AnotherUnknownPlugin'
